server/room-battle.ts

```diff
@@ -234,6 +234,10 @@ export class RoomBattleTimer {
 	start(requester?: User) {
 		const userid = requester ? requester.id : 'staff' as ID;
 		if (this.timerRequesters.has(userid)) return false;
+		if (this.battle.ended && requester) {
+			requester.sendTo(this.battle.roomid, `|inactiveoff|The timer can't be enabled after a battle has ended.`);
+			return false;
+		}
```

Comment:
This should probably also cut in if there's no `requester`:

```suggestion
		if (this.battle.ended) {
			requester?.sendTo(this.battle.roomid, `|inactiveoff|The timer can't be enabled after a battle has ended.`);
			return false;
		}
```

Reply:
Yep, this works great.

![image](https://user-images.githubusercontent.com/23667022/151717333-ad4a5b75-7ca8-490b-a239-0f3ddf20ca49.png)